.github/workflows/terraform-ci.yml

```
@@ -108,6 +108,10 @@ jobs:
         aws configure set role_arn "${{ secrets.AWS_ROLE_ARN }}"
         aws configure set region us-west-2
         aws configure set source_profile lambda_user
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v3
```

We should pin the GitHub Action for better security.
```suggestion
      uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd #v3.1.2
```